.circleci/config.yml

```diff
@@ -1661,10 +1664,11 @@ workflows:
           name: op-batcher-tests
           module: op-batcher
           requires: ["go-mod-download"]
-      - go-test:
+      - go-test-kurtosis:
           name: op-chain-ops-tests
           module: op-chain-ops
-          requires: ["go-mod-download"]
+          uses_artifacts: true
+          requires: ["contracts-bedrock-build"]
```

Comment:
Can we put the kurtosis tests, and maybe op-deployer as a whole, in its own package? Most op-chain-ops tests are local and simple, they don't require kurtosis.

Reply:
Sure. Will make OP deployer a standalone package.

Reply:
Actually - will separate it out as as standalone package later, since there's a PR stack on this one.